pkg/options/types.go

```diff
@@ -99,7 +99,7 @@ func (c CollectorSet) AsSlice() []string {
 
 // isEmpty() returns true if the length of the CollectorSet is zero.
 func (c CollectorSet) isEmpty() bool {
```

Comment:
`isEmpty()` is unused. Let's remove it for now :).

Reply:
Sounds good. I noticed this was unused, but wasn't sure whether to remove in case this was work in progress.

Reply:
Removed!